### PR TITLE
Add editMember command

### PR DESCRIPTION
**Pull request by cheeerynne**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/EditMemberCommandParser.java [hunk 1]
```diff
@@ -6,50 +6,51 @@
...
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE), pe);
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    EditMemberCommand.MESSAGE_USAGE), pe);
         }
 
-        EditPersonDescriptor editPersonDescriptor = new EditPersonDescriptor();
+        EditMemberCommand.EditMemberDescriptor editMemberDescriptor = new EditMemberDescriptor();
         if (argMultimap.getValue(PREFIX_NAME).isPresent()) {
-            editPersonDescriptor.setName(ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get()));
+            editMemberDescriptor.setName(ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get()));
```
Comment (zatkiller) [inline src/main/java/seedu/address/logic/parser/EditMemberCommandParser.java]: I thought names should not have prefix?

File: src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java [hunk 2]
```diff
@@ -12,56 +12,48 @@
...
 
-    private EditCommandParser parser = new EditCommandParser();
+    private EditMemberCommandParser parser = new EditMemberCommandParser();
 
     @Test
     public void parse_missingParts_failure() {
-        // no index specified
-        assertParseFailure(parser, VALID_NAME_AMY, MESSAGE_INVALID_FORMAT);
+        // no name specified
+        assertParseFailure(parser, "-p95652233", MESSAGE_INVALID_FORMAT);
```
Comment (zatkiller) [inline src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java]: Should we follow the format of the actual phone number -p 95652233 (with spacing)

File: src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java [hunk 3]
```diff
@@ -12,56 +12,48 @@
...
-        // zero index
-        assertParseFailure(parser, "0" + NAME_DESC_AMY, MESSAGE_INVALID_FORMAT);
-
         // invalid arguments being parsed as preamble
-        assertParseFailure(parser, "1 some random string", MESSAGE_INVALID_FORMAT);
+        assertParseFailure(parser, " ", MESSAGE_INVALID_FORMAT);
 
         // invalid prefix being parsed as preamble
-        assertParseFailure(parser, "1 i/ string", MESSAGE_INVALID_FORMAT);
+        assertParseFailure(parser, "/random", MESSAGE_INVALID_FORMAT);
```
Comment (zatkiller) [inline src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java]: I think we should follow our prefix convention using -. For e.g. -a

File: src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java [hunk 4]
```diff
@@ -12,56 +12,48 @@
...
     public void parse_invalidPreamble_failure() {
-        // negative index
-        assertParseFailure(parser, "-5" + NAME_DESC_AMY, MESSAGE_INVALID_FORMAT);
-
-        // zero index
-        assertParseFailure(parser, "0" + NAME_DESC_AMY, MESSAGE_INVALID_FORMAT);
-
         // invalid arguments being parsed as preamble
-        assertParseFailure(parser, "1 some random string", MESSAGE_INVALID_FORMAT);
+        assertParseFailure(parser, " ", MESSAGE_INVALID_FORMAT);
```
Comment (zatkiller) [inline src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java]: I think this should be changed to invalid name, try including some alphabets mixed with numbers

**Review by zatkiller — CHANGES_REQUESTED** [review]
Some nitpicks but rest looks good

File: src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java (hunk 4 above)
Comment (cheeerynne) [reply src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java]: Ok!

File: src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java (hunk 3 above)
Comment (cheeerynne) [reply src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java]: Ok sure!

File: src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java (hunk 2 above)
Comment (cheeerynne) [reply src/test/java/seedu/address/logic/parser/EditMemberCommandParserTest.java]: Sure!

**Review by zatkiller — APPROVED** [review]
LGTM

**Pull request merged**